### PR TITLE
Fixed date parsing of Error objects

### PR DESCRIPTION
**Pull request by ChrisAnn**
The dates being returned in the Error object are in an unexpected format which include milliseconds to 3 places.

Retrieval of errors now specifies the exact format.  Updated the MocedApi Error stubs to return a date format matching live and tests pass.

### CODE REVIEW

**Comment by friism** [comment]
Looks good, but the line-endings are changed on lines above the new stuff you introduce -- check out the diff: https://github.com/appharbor/AppHarbor.NET/pull/14/files

**Comment by ChrisAnn** [comment]
Ah, thanks.  I switched into Windows and mustn't have set autocrlf to false there.  

Fixed now.

File: AppHarbor.Test/Data/GetError.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿{
    "commit_id" : "foo",
-    "date" : "2012-03-05T15:01:42",
+	"date": "2012-06-30T11:05:29.647Z",
     "request_path" : "/",
```
Comment (friism) [inline AppHarbor.Test/Data/GetError.json:4]: please preserve the whitespace convention (spaces). You can view whitespace in Visual Studio by hitting ctrl+r, ctrl+w

File: AppHarbor.Test/Data/GetErrors.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ﻿[{
    "commit_id" : "foo",
-    "date" : "2012-03-05T15:01:11",
+	"date": "2012-06-30T11:05:29.647Z",
     "request_path" : "/",
```
Comment (friism) [inline AppHarbor.Test/Data/GetErrors.json:4]: please preserve the whitespace convention (spaces). You can view whitespace in Visual Studio by hitting ctrl+r, ctrl+w

**Comment by friism** [comment]
Closer inspection has revealed what I think to an error in the RestSharp dateparsing code. There's a fix here: https://github.com/restsharp/RestSharp/pull/302

**Comment by ChrisAnn** [comment]
Ah brilliant.  At the time I looked at the documentation for RestSharp it
wasn't listed as supported automatically, but I see that you decided to
change that :)

On 20 July 2012 00:32, Michael Friis <
reply@reply.github.com

**Pull request closed without merging**